src/Qir/Tools/Driver/QirCppDriverGenerator.cs

```diff
@@ -33,20 +33,15 @@ public async Task GenerateAsync(EntryPointOperation entryPointOperation, Stream
 
         public string GetCommandLineArguments(ExecutionInformation executionInformation)
         {
-            // Sort arguments by position.
-            var sortedArguments = executionInformation.EntryPoint.Parameters.OrderBy(arg => arg.Position);
-            var argumentBuilder = new StringBuilder();
-            foreach (var arg in sortedArguments)
+            string Argument(Parameter param)
```

Comment:
nit: naming to something more descriptive such as `{Map/Convert}ToArgument` or something similar.

Reply:
Hmm, I'm not sure that that's more descriptive... a function by definition converts its input to its output, so the name `Argument` should be enough to deduce that it converts a parameter to a string argument.

Reply:
I renamed it to `ToArgument`.